ql/src/test/org/apache/hadoop/hive/ql/TestTxnCommands2.java

```diff
@@ -1335,6 +1319,7 @@ public void testInitiatorWithMinorCompactionForInsertOnlyTable() throws Exceptio
    * Make sure there's no FileSystem$Cache$Key leak due to UGI use
    * @throws Exception
    */
+  @Ignore("FileSystem.closeAllForUGI is never called from Compaction related threads")
```

Comment:
if that is the case, why do we need this test at all?

Reply:
It makes sense to remove it. Done

Reply:
👍